### PR TITLE
Adding label to the pod template

### PR DESCRIPTION
**Pull request by Tariqs**
Adding label to the pod template because its easier for user to understand that label can be added easily. I struggled with this a bit so I am adding it for future readers.

### CODE REVIEW

File: content/en/docs/concepts/workloads/pods/_index.md [hunk 1]
```diff
@@ -195,6 +195,9 @@ metadata:
 spec:
   template:
     # This is the pod template
+    metadata:
+      labels:             
+        id: awesome-label
```
Comment (tengqm) [inline content/en/docs/concepts/workloads/pods/_index.md:200]: label should be something meaningful, useful ...

Comment (Tariqs) [reply content/en/docs/concepts/workloads/pods/_index.md]: I have updated the label. Please check.

**Review by Ritikaa96 — CHANGES_REQUESTED** [review]

**Comment by a-mccarthy** [comment]
Hi @Tariqs are you able to address the comments on this PR? thanks!

**Comment by tengqm** [comment]
/close
due to no response from author for many months.
Feel free to reopen.

**Pull request closed without merging**